pages/press.md

```diff
@@ -10,6 +10,15 @@ Are you looking to get a speaker or interview about Solid? Contact us on [press@
 
 ### 2020
 
+{%
+  include talk.html
+    date="2020-09-03"
+    title="Solid World September"
+    website="https://ipfs.io/ipfs/QmSH1aut6uikU8EEqRN3FLY6fXqVHcSJcw1JBqBgWvYLe9"
```

Comment:
Hmm, this doesn't load for me, and I don't think Solid World uses IPFS to host their website anyway? For consistency with the older editions, we can use the EventBrite link, although there doesn't seem to be too much there anymore:

```suggestion
    website="https://www.eventbrite.com/e/solid-world-tickets-115477011851"
```

Alternatively, we can just leave out the `website` mentions for Solid World editions altogether?

Reply:
```suggestion
    website="https://solidproject.org/solid-world-september"
```

Reply:
I like that a lot, and I think from the point of view of solidproject.org it would be fine to host all the Solid World information, but in the end that is up to the organisers of Solid World, and last I heard they don't have the capacity to go through the website update process, so we would be displaying outdated information :(